### DISCUSSION TITLE
HMG With Storage Container Meth Ammunition Causes Error On Hull Hit

### DISCUSSION THREAD

**Discussion by Kaihenthe** [FakeFishGames/Barotrauma#15351] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### When using a HMG with meth inside of a container as HMG ammunition, The game attempts to apply status effects when it hits a wall (doors and characters don't produce this error), The game shouldn't be attempting to apply meth to a submarine hull.
Also i decided to disabled all mods to test it and it still happened, I don't have client lua installed
![image](https://github.com/user-attachments/assets/a0d2f255-a411-41e3-9bf1-685f02fb99dd)
**Note, I did not test this in multiplayer, Only Singleplayer and Submarine Editor**

### Reproduction steps

**Step to reproduce**:
1: Get any submarine
2: Get a HMG
3: Get a Storage Container With tag "hmgammo"
4: Put 1 or more Methamphetamine items inside the Storage Container _(I attempted to use morphine to see did it make the error aswell but it dosen't)_
5: Fire at a submarine hull, The f3 console should appear with the error on hit with inside walls or hull.

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Single player

### Version

v1.7.7.0 (Winter Update)

### Which operating system did you encounter this bug on?

Windows

### Relevant error messages and crash reports

```shell
Failed to set  the value of the property "Tags" of "Large Horizontal Inside Wall" to medical (Type "ImmutableHashSet1" not supported)
```

**Comment by Kaihenthe** [comment]
My character is lacking the skills to properly wield the gun, Haven't tested if that's the issue or not, I am not sure every gun that has this issue but shotgun also does when directly loaded with meth as the ammo

**Comment by Jasontti** [comment]
Tested the fix for this and issue is fixed.
Closing discussion as resolved.
